kornia/geometry/epipolar/fundamental.py

```diff
@@ -128,6 +331,32 @@ def find_fundamental(
     return normalize_transformation(F_est)
 
 
+def find_fundamental(
+    points1: torch.Tensor, points2: torch.Tensor, weights: Optional[torch.Tensor] = None, method: str = "8POINT"
```

Comment:
https://mypy.readthedocs.io/en/stable/literal_types.html
```suggestion
    points1: torch.Tensor, points2: torch.Tensor, weights: Optional[torch.Tensor] = None, method: Literal['8POINT', '7POINT'] = "8POINT"
```